### PR TITLE
Created a directive to allow homepage element to set it's height to the computed containerHeight

### PR DESCRIPTION
**Pull request by pwpatton**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
W/o a height on homepage we weren't able to use position sticky on any element before the home page. With it's element height computed correctly the container now autoheights to the correct size and the sticky tabs we have stick at their correct position.

**Steps necessary to review your pull request (required)**:
- open ng app
- H -> HomePage Scenario A
- verify in debugger that div.homepage is getting it's height reset on each resize of the homepage container.

### CODE REVIEW

File: projects/ids-enterprise-ng/src/lib/homepage/soho-homepage-sizer.directive.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import {
+  AfterViewInit,
+  Directive,
+  ElementRef, HostBinding,
+  NgZone, OnDestroy
+} from '@angular/core';
+
+/**
+ * Angular Wrapper for the SohoAlert Directive.
```
Comment (fitzorama) [inline projects/ids-enterprise-ng/src/lib/homepage/soho-homepage-sizer.directive.ts]: SohoSizer

**Comment by tmcconechy** [comment]
Are we waiting on things here?

**Comment by pwpatton** [comment]
yea, waiting for an EP build with the dependent homepage.js code.

**Comment by tmcconechy** [comment]
OK i made a `4.21.0-dev.20190809` and pushed it on this branch. Give it a test...

**Pull request merged**